dev/com.ibm.ws.app.manager/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -85,4 +85,4 @@ classloader.name=Classloader
 classloader.desc=Defines the settings for an application classloader.
 
 appmgmt.expandLocation=Expand location
-appmgmt.expandLocation.desc=The expandLocation attribute enables a user to set the location where the autoExpanded application is located. The default location is ${server.config.dir}/apps/expanded/.
+appmgmt.expandLocation.desc=The expandLocation attribute enables a user to set the location where the autoExpanded application is located. The default location is ${server.config.dir}/apps/expanded/. The same expandLocation location cannot be shared between multiple servers.
```

Comment:
Change "between" to "among".

Reply:
I've made this change and kicked off a build.